src/chains/ethereum/ethereum/src/database.ts

```diff
@@ -1,14 +1,13 @@
-import { AbstractLevelDOWN } from "abstract-leveldown";
+import type { AbstractLevelDOWN } from "abstract-leveldown";
 import Emittery from "emittery";
 import { dir, setGracefulCleanup } from "tmp-promise";
-// import levelup, { LevelUp } from "levelup";
 import Blockchain from "./blockchain";
 import { EthereumInternalOptions } from "@ganache/ethereum-options";
-import levelup from "levelup";
-import type { LevelUp } from "levelup";
-import leveldown from "leveldown";
 import sub from "subleveldown";
 import encode from "encoding-down";
+import leveldown from "leveldown";
+import type { LevelUp } from "levelup";
+const levelup = require("levelup");
```

Comment:
The latest LevelUp types are exported at the wrong location so we can't use the runtime and the types in the same import. See: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/59192